### DISCUSSION TITLE
Achievements don't exist

### DISCUSSION THREAD

**Discussion by Amgad-Abdallah-Mahmoud** [community/community#123451] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I don't have achievements section in my GitHub, and when I go to https://github.com/Amgad-Abdallah-Mahmoud?tab=achievements it gives me error 404 not found, can you please help me?

**Comment by IgorD-lab** [comment, marked as answer]
See https://github.com/orgs/community/discussions/50552 and https://github.com/orgs/community/discussions/56662.

**Reply by Amgad-Abdallah-Mahmoud** [reply]
I have tried everything here but still doesn't work

**Reply by IgorD-lab** [reply]
Make sure your achievements arent attacked to your private repositories not much else can be done without contacting the support.

**Reply by Amgad-Abdallah-Mahmoud** [reply]
Okay thank you very much for your quick responses!
